.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@abhiramm7, please remove this file.  We would like to keep all build artifacts out of this project.  an alternative to `git add -a` is `git add -u` which only stages the files that are currently under version control.